includes/Features/Abilities_Explorer/Admin_Page.php

```diff
@@ -0,0 +1,552 @@
+<?php
+/**
+ * Admin Page Class
+ *
+ * Handles admin menu, pages, and UI rendering.
+ *
+ * @package WordPress\AI\Features\Abilities_Explorer
+ * @since 0.1.0
+ */
+
+namespace WordPress\AI\Features\Abilities_Explorer;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+/**
+ * Admin Page Class
+ *
+ * Manages the admin interface for Abilities Explorer.
+ *
+ * @since 0.1.0
+ */
+class Admin_Page {
+
+	/**
+	 * Feature directory path.
+	 *
+	 * @since 0.1.0
+	 * @var string
+	 */
+	private $feature_dir;
+
+	/**
+	 * Feature directory URL.
+	 *
+	 * @since 0.1.0
+	 * @var string
+	 */
+	private $feature_url;
+
+	/**
+	 * Constructor.
+	 *
+	 * @since 0.1.0
+	 *
+	 * @param string $feature_dir Feature directory path.
+	 * @param string $feature_url Feature directory URL.
+	 */
+	public function __construct( $feature_dir, $feature_url ) {
+		$this->feature_dir = $feature_dir;
+		$this->feature_url = $feature_url;
+	}
+
+	/**
+	 * Initialize admin functionality.
+	 *
+	 * @since 0.1.0
+	 */
+	public function init() {
+		add_action( 'admin_menu', array( $this, 'add_admin_menu' ) );
+		add_action( 'admin_enqueue_scripts', array( $this, 'enqueue_assets' ) );
+		add_action( 'wp_ajax_ai_ability_explorer_invoke', array( $this, 'ajax_invoke_ability' ) );
+	}
+
+	/**
+	 * Add admin menu item.
+	 *
+	 * @since 0.1.0
+	 */
+	public function add_admin_menu() {
+		// Add top-level menu
+		add_menu_page(
+			__( 'Abilities Explorer', 'ai' ),
+			__( 'Abilities', 'ai' ),
+			'manage_options',
+			'ai-abilities-explorer',
+			array( $this, 'render_page' ),
+			'dashicons-superhero',
+			30
+		);
+	}
+
+	/**
+	 * Enqueue admin assets.
+	 *
+	 * @since 0.1.0
+	 *
+	 * @param string $hook Current admin page hook.
+	 */
+	public function enqueue_assets( $hook ) {
+		// Only load on the Abilities Explorer page
+		if ( 'toplevel_page_ai-abilities-explorer' !== $hook ) {
+			return;
+		}
+
+		// Enqueue styles
+		wp_enqueue_style(
+			'ai-ability-explorer-admin',
+			$this->feature_url . '/assets/css/abilities-explorer.css',
+			array(),
+			Abilities_Explorer::VERSION
+		);
+
+		// Enqueue scripts
+		wp_enqueue_script(
+			'ai-ability-explorer-admin',
+			$this->feature_url . '/assets/js/abilities-explorer.js',
+			array( 'jquery' ),
+			Abilities_Explorer::VERSION,
+			true
+		);
+
+		// Localize script
+		wp_localize_script(
```

Comment:
I'm guessing we're setting nonces here for some later admin-ajax implementation. It would be really cool if we used the Abilities API's JavaScript client (dogfooding our own API) instead.